junit-runtime/src/main/scala/org/junit/internal/InexactComparisonCriteria.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Ported from https://github.com/junit-team/junit
+ */
+package org.junit.internal
+
+import org.junit.Assert
+
+class InexactComparisonCriteria private(val fDelta: AnyRef)
+    extends ComparisonCriteria {
+
+  def this(delta: Double) =
+    this(delta.asInstanceOf[AnyRef])
+
+  def this(delta: Float) =
+    this(delta.asInstanceOf[AnyRef])
+
+  override protected def assertElementsEqual(expected: AnyRef,
+      actual: AnyRef): Unit = {
+    expected match {
+      case expected: java.lang.Double =>
```

Comment:
This will always be true in Scala.js, even if `expected` was originally a `Float`.

I guess you can drop the pattern match and always cast the three arguments to `Double`.